core/nullDC.cpp

```diff
@@ -297,6 +297,7 @@ void LoadSettings()
 	settings.aica.LimitFPS			= cfgLoadInt("config", "aica.LimitFPS", 1);
 	settings.aica.NoBatch			= cfgLoadInt("config", "aica.NoBatch", 0);
 	settings.aica.NoSound			= cfgLoadInt("config", "aica.NoSound", 0);
+	settings.aica.OldSyncronousDma = cfgLoadBool("config", "aica.OldSyncronousDma", false);
```

Comment:
Indentation.

As much as I dislike the tabular layout it currently is, it, well, currently is and should therefor stay as it is or be reformatted as a whole.

Reply:
Oh well, I just switched the diff to a bigger viewpoint and saw that there already are many different identations here...